### PR TITLE
APEXCORE-753 Add mailing lists, SCM and issue tracking information to POM

### PR DESCRIPTION
**Pull request by vrozov**
@tweise Please review and merge

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,6 +50,54 @@
...
+        <otherArchive>https://lists.apache.org/list.html?dev@apex.apache.org</otherArchive>
+        <otherArchive>https://s.apache.org/apex-dev</otherArchive>
+      </otherArchives>
+    </mailingList>
+    <mailingList>
+      <name>Apex User</name>
+      <subscribe>users-subscribe@apex.apache.org</subscribe>
+      <unsubscribe>users-unsubscribe@apex.apache.org</unsubscribe>
+      <post>users@apex.apache.org</post>
+      <archive>http://mail-archives.apache.org/mod_mbox/apex-users/</archive>
```
Comment (tweise) [inline pom.xml]: lists is the primary archive, mod_mbox is deprecated AFAIK

Comment (vrozov) [reply pom.xml]: http://apex.apache.org/community.html#mailing-lists still points to mod_mbox as a primary archive. Should we change both?

Comment (tweise) [reply pom.xml]: Yes, that would be good to change also.

Comment (vrozov) [reply pom.xml]: changed to lists and rebased, please review

**Comment by tweise** [comment]
LGTM

**Pull request merged**